### ISSUE TITLE
RangeSlider's RangeSliderThumb is rendered incorrectly when within Accordion

### ISSUE THREAD

**Issue by dufia** [chakra-ui/chakra-ui#5255]
### Description

When you place RangeSlider in the Accordion the RangeSliderThumb is rendered too much to the right

### Link to Reproduction

https://codesandbox.io/s/homepage-forked-3v00t?file=/src/App.tsx:507-523

### Steps to reproduce

1. Go to [link](https://codesandbox.io/s/homepage-forked-3v00t?file=/src/App.tsx:507-523)
2. If you look **very** closely you can see that on the second RangeSlider, RangeSliderThumb are offset to the right a little bit

<img width="716" alt="Screenshot 2021-12-17 at 18 05 37" src="https://user-images.githubusercontent.com/5569649/146582262-7de1e4f1-f965-46ef-9010-37de7bc9a735.png">

### Chakra UI Version

1.7.3

### Browser

Chrome 96.0.4664.93

### Operating System

- [X] macOS
- [ ] Windows
- [ ] Linux

**Comment by primos63** [comment]
If you align the standalone `RangeSlider` with the one in the `AccordionPanel` you'll see that the left and right thumb are to the right of where those in the standalone slider are. This is because of the calculation that is done to position the thumbs relies on the `offsetWidth` of the slider thumb. Before the panel is expanded the `offsetWidth` is 0 and this causes them to be placed farther to the right.

This only works when the `Accordion`'s internal state prop, `isExpanded`, is used. The `RangeSlider` needs to be visible to be able to get the offsetWidth.
```jsx
  <Accordion allowToggle={true} borderX="1px" borderTop="1px">
    <AccordionItem>
      {({ isExpanded }) => (
        <>
          <AccordionButton>
            <Box flex="1" textAlign="left">
              Click to expand
            </Box>
            <AccordionIcon />
          </AccordionButton>
          <AccordionPanel bg="cyan.100">
            {isExpanded ? <CustRangeSlider /> : null}
          </AccordionPanel>
        </>
      )}
    </AccordionItem>
  </Accordion>
```

My [CodeSandbox example](https://codesandbox.io/s/homepage-forked-uy0ci?file=/src/App.tsx) shows the RangeSlider with my proposed change.

**Comment by dufia** [comment]
Thanks @primos63

**Comment by dufia** [comment]
@segunadebayo was it fixed? Cheers

**Comment by dufia** [comment]
@segunadebayo?

**Comment by jdddog** [comment]
@dufia and @segunadebayo this isn't fixed in v1.8.8.
